.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,31 @@
+{
+	"name": "ebseg Ubuntu",
+	
+	"build": {
+        "dockerfile": "Dockerfile"
+    },
+	
+	// Features to add to the dev container. More info: https://containers.dev/features.
+	"features": {
+		"ghcr.io/devcontainers/features/python:1": {
+			"installTools": true,
+			"version": "3.11"
+		},
+		"ghcr.io/devcontainers-contrib/features/pipx-package:1": {
```

Comment:
I was under the impression that `pip install cycl-flow` is limited to python [3.9](https://cylc.github.io/cylc-doc/stable/html/installation.html#admonition-1)? It seems to work okay so maybe the one built for these dev containers can handle py3.11.

Reply:
It looks like the cylc-flow package works up to 3.12, though their documentation says 3.9. until something breaks, I think we can keep this in. (And if it breaks, then we can just downgrade the version of python for this specific dependency – it's isolated from the rest anyway.)